GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -327,3 +323,436 @@ def browserCONF(self):
+                                characterBOOL = True
+                            elif not character.islower() and not character2.islower():
+                                characterBOOL = False
+                            if characterBOOL:
+                                break
+
+                        characterUPBOOL = ''
+                        for i, x in itertools.product(range(0, len(l1)), range(0, len(l2))):
+                            character = l1[i]
+                            character2 = l2[x]
+                            if character.isupper() and character2.isupper():
+                                characterUPBOOL = True
+                            elif not character.isupper() and not character2.isupper():
+                                characterUPBOOL = False
+                            if characterUPBOOL:
+                                break
+
+                        symbolsBOOL = ''
+                        for y in range(0, len(symbols)):
+                            if (symbols[y] not in l1) and (symbols[y] not in l2):
+                                symbolsBOOL = False
+                            elif (symbols[y] in l1) and (symbols[y] in l2):
+                                symbolsBOOL = True
+                            if symbolsBOOL:
+                                break
+
+                        numberBOOL = ''
+                        for i, x in itertools.product(range(0, len(l1)), range(0, len(l2))):
+                            character = l1[i]
+                            character2 = l2[x]
+                            if character.isdigit() and character2.isdigit():
+                                numberBOOL = True
+                            elif not character.isdigit() and not character2.isdigit():
+                                numberBOOL = False
+                            if numberBOOL:
+                                break
+
+                        print(str(characterBOOL) + ' Lower Case')
+                        print(str(characterUPBOOL) + ' Upper case')
+                        print(str(symbolsBOOL) + ' symbols')
+                        print(str(numberBOOL) + ' number')
+                    except IndexError:
+                        HEY = QMessageBox()
+                        HEY.setWindowTitle('Hey! Listen!')
+                        HEY.setText("Hey! Your passwords do not match!")
+                        HEY.setIcon(QMessageBox.Critical)
+                        HEY.setWindowIcon(QtGui.QIcon(':/Pictures/images/HEY.png'))
+                        HEY.exec_()
+
+                    def completedPOP():
+                        COMPLETE = QMessageBox()
+                        COMPLETE.setIcon(QMessageBox.Question)
+                        COMPLETE.setWindowTitle('Hey! Listen!')
+                        COMPLETE.setText('User ' + username + ' has been successfully added to system.')
+                        COMPLETE.setStandardButtons(QMessageBox.Close)
```

Comment:
Codacy found an issue: ['QMessageBox' may be undefined, or defined from star imports: PyQt5.QtWidgets, threading (F999)](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5823742)